finality-aleph/src/sync/ticker.rs

```diff
@@ -1,39 +1,36 @@
 use tokio::time::{sleep, Duration, Instant};
 
 /// This struct is used for rate limiting as an on-demand ticker. It can be used for ticking
-/// at least once `max_timeout` but not more than once every `min_timeout`.
-/// Example usage would be to use `wait` method in main select loop and `try_tick` whenever
-/// you would like to tick sooner in another branch of select.
+/// at most after `max_timeout` but no sooner than after `min_timeout`.
+/// Example usage would be to use the `wait` method in main select loop and
+/// `try_tick` whenever you would like to tick sooner in another branch of select,
+/// resetting whenever the rate limited action actually occurs.
 pub struct Ticker {
-    last_tick: Instant,
+    last_reset: Instant,
```

Comment:
while we are changing this could we also rename current_timeout to some better name :<